lang/ko.js

```diff
@@ -26,13 +26,13 @@ export default {
 	"components.form-element.defaultError": "{label}이(가) 잘못되었습니다.",
 	"components.form-element.defaultFieldLabel": "필드",
 	"components.form-element.input.email.typeMismatch": "이메일이 유효하지 않습니다.",
-	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {Number must be greater than {min} and less than {max}.} other {Number must be greater than {min} and less than or equal to {max}.}}} other {{maxExclusive, select, true {Number must be greater than or equal to {min} and less than {max}.} other {Number must be greater than or equal to {min} and less than or equal to {max}.}}}}",
-	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {숫자가 보다 작음 {max}여야 합니다.} other {숫자가 작거나 같음 {max}여야 합니다.}}",
-	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {숫자가 보다 큼 {min}여야 합니다.} other {숫자가 크거나 같음 {min}여야 합니다.}}",
+	"components.form-element.input.number.rangeError": "{minExclusive, select, true {{maxExclusive, select, true {숫자는 {min}보다 크고 {max}보다 작아야 합니다.} other {숫자는 {min}보다 크고 {max}보다 작거나 같아야 합니다.}}} other {{maxExclusive, select, true {숫자는 {min}보다 크거나 같고 {max}보다 작아야 합니다.} other {숫자는 {min}보다 크거나 같고 {max}보다 작거나 같아야 합니다.}}}}",
+	"components.form-element.input.number.rangeOverflow": "{maxExclusive, select, true {숫자는 {max}보다 작아야 합니다.} other {숫자는 {max}보다 작거나 같아야 합니다.}}",
+	"components.form-element.input.number.rangeUnderflow": "{minExclusive, select, true {숫자는 {min}보다 커야 합니다.} other {숫자는 {min}보다 크거나 같아야 합니다.}",
```

Comment:
Before `The number must be greater than {min}.` After is same.

Before `Number must be greater than or equal to {min}.` After `The number must be greater than or equal to {min}.` After seems correct/consistent (minus missing bracket)